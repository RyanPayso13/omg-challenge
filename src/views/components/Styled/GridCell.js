import styled from 'styled-components';

export const GridCell = styled.div`
    position: relative;
    box-shadow: rgba(0, 0, 0, 0.25) 0px 2px 6px;
`;