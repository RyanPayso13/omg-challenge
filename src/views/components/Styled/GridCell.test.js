import React from 'react';
import { render } from '@testing-library/react';
import { GridCell } from './GridCell.js';

describe('<GridCell />', () => {

    it('should work', () => {
        const { container } = render(<GridCell />);
        expect(container.firstChild).toMatchSnapshot();    
    });

});
