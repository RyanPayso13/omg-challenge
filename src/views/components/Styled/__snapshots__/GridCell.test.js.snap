// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridCell /> should work 1`] = `
.c0 {
  position: relative;
  box-shadow: rgba(0,0,0,0.25) 0px 2px 6px;
}

<div
  class="c0"
/>
`;
